web/src/people.ts

```diff
@@ -112,6 +112,45 @@ export function init(): void {
 // WE INITIALIZE DATA STRUCTURES HERE!
 init();
 
+// A function that sorts Email according to the user's full name
+export function sortEmails(emails: string[], user_ids: number[]): string[] {
```

Comment:
We do not use camelCase for naming functions and variables, instead we use snake_case.